### DISCUSSION TITLE
Notification shortcut

### DISCUSSION THREAD

**Discussion by whdmike** [iSoron/uhabits#789] [Feature Requests]
I am happy user of habits. My suggestion would be to change notification shortcut - when notification appears on a screen or notification area, and I press it, I am redirected to editing section of the habit, not to the List where I can mark habit done. I find this very unpractical. Using android, galaxy S21.

**Comment by hiqua** [comment]
But you already have `Yes / No / Later` right? So if you want to mark it done, wouldn't you use `Yes` instead?

**Reply by whdmike** [reply]
Thank you, it is also a way. I didn't know about it. Thou I would prefer change of shortcut.
